src/site/content/en/blog/pwas-on-oculus-2/index.md

```diff
@@ -0,0 +1,348 @@
+| `ovr_package_name`      | no       | Sets the package name of the APK generated for the PWA. This must be in reverse domain name notation, e.g., `"com.company.app.pwa"`. If not set, developers must provide a package name to the command line tool with the (then required) parameter `--package-name`.                                                                                                                                                                |
+| `ovr_multi_tab_enabled` | no       | If `true`, this boolean field will give the PWA a tab bar similar to Oculus Browser. In multi-tab PWAs, internal links that target a new tab (`target="_new"` or `target="_blank"`) will open in new tabs within the PWA window. This differs from single-tab PWAs where such links would open to a Oculus Browser window. **This feature is currently being standardized as [Tabbed Application Mode](/tabbed-application-mode/).** |
+| `ovr_scope_extensions`  | no       | This field allows a PWA to include more web pages within the scope of the web application. It consists of a JSON dictionary containing extension URLs or wildcard patterns. **This feature is currently being standardized as [Scope Extensions for Web Apps](https://github.com/WICG/manifest-incubations/blob/gh-pages/scope_extensions-explainer.md).**                                                                           |
+
+### Packaging PWAs with the command line tool
+
+To sideload PWAs onto the Oculus Quest&nbsp;2, developers must use a
+[command line tool](https://developer.oculus.com/documentation/web/pwa-packaging/) called
+`ovr-platform-util` with the `create-pwa` command to generate the APK. The output file name can be
+set via the `-o` parameter, the Android SDK to use needs to be set via the `--android-sdk`
+parameter. Assuming a manifest file called `manifest.json` in the present working directory, the
+`--manifest-content-file` parameter helps the tool to determine the relevant configuration fields
+from the Web App Manifest. Alternatively, developers can point the tool at the live URL of a
+manifest via the `--web-manifest-url` parameter. To leave the manifest as pure as possible, the
+`--package-name` parameter can be used with a value in reverse domain name notation (e.g.,
+`com.company.app.pwa`), rather than adding the proprietary `ovr_package_name` field to the manifest.
+
+```bash
+ovr-platform-util create-pwa -o output.apk --android-sdk ~/bin/android-10 --manifest-content-file manifest.json --package-name com.company.app.pwa
+```
+
+### Installing PWAs with `adb`
+
+After creating the APK, developers can sideload the PWA with the `adb` command when the Oculus
+Quest&nbsp;2 device is connected via USB and when the connected computer is trusted. Sideloaded apps
```

Comment:
>when the connected computer is trusted

What does this mean? How do I learn if my computer is trusted? If this isn't the place to explain this, maybe provide links.

Reply:
The link to the relevant docs is there, but the trust step was missing. Added in https://github.com/GoogleChrome/developer.chrome.com/pull/1867.